packages/rocketchat-slashcommands-tokenpass/client/tokenpass-add-address.js

```diff
@@ -0,0 +1,8 @@
+Meteor.call('checkTokenpassOAuthEnabled', (error, result) => {
```

Comment:
calling `checkTokenpassOAuthEnabled` every time before adding a slash command is not a good implementation. I would recommend checking it once, then adding all slash commands at once.

but for this case it is even worse because if you enable the tokenpass integration after a while, slash commands will not be available since the integration was not enable on startup.

Reply:
In this case, we are talking about startup on client side after the Tokenpass integration was enabled, so I think that's it (adding all slash commands at once on check). Can you see another way?